go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/cloudflare/gokey v0.1.0
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
 	github.com/edgexfoundry/go-mod-bootstrap v0.0.11
-	github.com/edgexfoundry/go-mod-core-contracts v0.1.37
+	github.com/edgexfoundry/go-mod-core-contracts v0.1.41
```

Comment:
Now that all of the other changes are in `go-mod-core-contracts` should we update to `v0.1.42`?

Reply:
I think there's additional refactoring work in-flight related to this issue (https://github.com/edgexfoundry/go-mod-core-contracts/issues/196) to consolidate common code into the underlying client implementation.  Once that's complete, there's an additional follow-on issue (https://github.com/edgexfoundry/go-mod-core-contracts/issues/204) that should be worked before actually updating edgex-go to the latest (that would include those outstanding changes).